src/mocks/msw.js

```diff
@@ -0,0 +1,32 @@
+const msw = await new Promise((resolve,reject)=>
```

Comment:
This is a secret sause of MSW integration in `@web/test-runner` invoked by import maps.
 
`StoryBook` does not need it as uses WebPack builder.